components/board/numato/board.go

```diff
@@ -104,7 +104,7 @@ type numatoBoard struct {
 	resource.Named
 	resource.AlwaysRebuild
 	pins    int
-	analogs map[string]board.Analog
+	analogs map[string]*pinwrappers.AnalogSmoother
```

Comment:
Same comment here as on the pi board: this code will be easier to test and easier to use if we store the interface instead of a specific implementation. I'd prefer this go back to `board.Analog`, and have `pinwrappers.AnalogSmoother` implement that interface.

Reply:
that's no longer possible since we're removing the CLose function. I would rather call that smoother directly since that is what is both fulfilling the interface in this driver and fulfilling the functionality of an analog reader on the two boards without one.